src/main/java/hng_java_boilerplate/user/enums/Role.java

```diff
@@ -1,5 +1,5 @@
 package hng_java_boilerplate.user.enums;
 
 public enum Role {
-    ROLE_USER, ROLE_ADMIN
+    ROLE_USER, ROLE_ADMIN, ROLE_SUPER_ADMIN
```

Comment:
what's the difference between admin and super_admin?

Reply:
Super admin can assign admin roles to other users.

Reply:
Can do things other admins can't do.